wordpress-mobile/WordPress-Android
# Removes - unused methods and classess using Android Studio Tool - Part 2

Fixes #

## Description 
This PR removes unused code from the Repo using Android Tool. `Code → Analyze → Run inspection by name..., type Unused declaration`

2nd Part of https://github.com/wordpress-mobile/WordPress-Android/pull/19788

<img width="547" alt="Screenshot 2023-12-26 at 10 31 26 AM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/17463767/52876d94-a8e6-4879-8aea-3a0e6f218058">


## To Test:
1. Smoke test the JP And WP app and verify that everything is working as expected 
2. Verify that the classess, methods and files deleted are unused. 


## Regression Notes

1. Potential unintended areas of impact
App is not working as expected

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and Smoke testing 

4. What automated tests I added (or what prevented me from doing so)
Nothing

PR submission checklist:

UI Changes testing checklist:
